change/@fluentui-react-menu-0df86758-7786-4a98-b39d-3ac0f52167dd.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Merge branch 'master' into feature/focus-indicator",
```

Comment:
After you will restore package bumps please remove unnecessary change files or fix comments in them as they are used to generate changelogs.